client/signup/utils.js

```diff
@@ -224,3 +224,8 @@ export const clearSignupDestinationCookie = () => {
 
 	document.cookie = cookie.serialize( 'wpcom_signup_complete_destination', '', options );
 };
+
+export const shouldForceLogin = flowName => {
+	const flow = flows.getFlow( flowName );
+	return !! flow.forceLogin;
```

Comment:
Unfortunately, getFlow will return `undefined` in cases where a flow isn't found:
```js
// in config/flows,js - getFlow
// if the flow couldn't be found, return early
if ( ! flow ) {
  return flow;
}
```
Because of this I think we should either do `get( flow, 'forceLogin' )` to let lodash `get` the property safely, or `!! flow && flow.forceLogin`, so that we can be sure not to get a `TypeError` here

Reply:
ah well spotted, I'll change this.

Reply:
Awesome, thanks for that @seear 👌